src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -1212,17 +1212,26 @@ private void registerKeyStoreProviderOnConnection(String keyStoreAuth, String ke
             String keyStoreLocation) throws SQLServerException {
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.coding.HiddenFieldCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java L1212

&apos;keyStoreLocation&apos; hides a field.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1222268675>*